client/core/lib/DevClient.ts

```diff
@@ -1,73 +1,83 @@
-import { DynamicoClient, Options, Dependencies } from '.';
+import { DynamicoClient, Options, Dependencies, InitOptions } from '.';
 import { NoopStorage } from './utils/NoopStorage';
 
 export interface DevOptions {
   dependencies: {
     versions: Dependencies;
     resolvers: Record<string, any>;
   };
-  callback: Function;
   interval?: number;
   urlOverride?: string;
 }
 
-export class DynamicoDevClient extends DynamicoClient {
```

Comment:
not sure I understand why is there a need to remove the inheritance here? wouldn't it work just the same?

Reply:
Because the api is different so inheritance doesn't make sense.
You said you used inheritance because you didn't want to duplicate code/logic, and I accomplished that by just using the regular client internally